### ISSUE TITLE
Compile models from loopback into es-mappings automatically

### ISSUE THREAD

**Issue by pulkitsinghal** [strongloop-community/loopback-connector-elastic-search#12]

**Comment by santoshbaggam** [comment]
Hi @pulkitsinghal, would like to know if this is already in place or this is still under development.

**Comment by santoshbaggam** [comment]
Definitely. Is there any medium to communicate with you so that I get to know the framework and the conventions to create a PR. Let me know, thanks.

**Comment by aquid** [comment]
Hi @santoshbaggam, @pulkitsinghal is traveling now and will get back to you soon. In the meantime, you can try to look over the code yourself. If you have any specific questions then our community gitter chatroom can help you (https://gitter.im/strongloop-community/loopback-connector-elastic-search). Please post your questions there and someone from the community might help you.
